.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug report
+description: File a bug report
+labels: ["bug"]
+
+body:
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce the bug.
+      value: |
+        1.
+        2.
+        3.
+        4.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: What should have happened.
+      placeholder: "Example: The validation should have caught the issue."
+    validations:
+      required: true
+
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current behaviour
+      description: What actually happened. If you were given an error, please provide it here.
+      placeholder: "Example: I received an `APIError`."
+    validations:
+      required: true
+
+  - type: textarea
+    id: version
+    attributes:
+      label: severn version
+      description: The version of severn you're using. Obtainable though `python -m severn`.
```

Comment:
How do we want to display Severn? As Severn, or severn? I'm thinking the former, but open to discussion.

Reply:
I agree, probably Severn - updated them to be capitalized.